### DISCUSSION TITLE
How to schedule cron for seconds

### DISCUSSION THREAD

**Discussion by vedprakash2017** [deta/cloud-docs#572] [Help]
How to schedule cron for seconds?

**Comment by sudo-subg** [comment]
You can't. https://docs.deta.sh/docs/micros/cron#set-cron
> Limits
>- Cron expressions that lead to rates faster than 1 minute are not supported.

**Reply by vedprakash2017** [reply]
thanks for the reference is there any alternative any way to do so?

**Reply by el-garro** [reply]
No, unless you try to chain micros, and that's probably a no-no.
